resources/testharness.js

```diff
@@ -568,7 +568,21 @@ policies and contribution forms [3].
         var test_name = name ? name : test_environment.next_default_test_name();
         var test_obj = new Test(test_name, properties);
         if (func) {
-            test_obj.step(func, test_obj, test_obj);
+            var value = test_obj.step(func, test_obj, test_obj);
```

Comment:
So we're only doing this for the initial step, not any later ones. Can we have a comment to explain why?

Reply:
Added, PTAL.